pkgs/build-support/build-fhsenv-bubblewrap/default.nix

```diff
@@ -124,6 +124,8 @@ let
         "ssl/certs"
         "ca-certificates"
         "pki"
+        # Custom dconf profiles
+        "dconf/profile"
```

Comment:
Might also need to mount `dconf/db`. For example, the `ibus` profile refers to a database in there.

Reply:
Good point, maybe the whole dconf directory should be mounted? I'm not aware of anything that could be in the directoy that shouldnt be mounted.